en/docs/extended_topics/model_mixed_parallel.md

```diff
@@ -20,47 +20,47 @@ We need to set up a simple multi-layer network first and use this network to dis
 
-* The input of layer 1 is `Data 1` matrix and `Model 1` matrix. Then apply `op` and get `output`.
+* The input of layer 1 is `Data 1` matrix and `Model 1` matrix. We apply `op` and get `output`.
 
 * The layer 2 is `output` layer and `Data 2` is the output of network. Of course, it can play as input in a deeper network.
 
 In `consistent` view, OneFlow supports the data parallelism, model parallelism and hybrid parallelism. We will introduce them in order but hybrid parallelism is the key point.
 
-## The characteristics of parallelism in consistent view
+## The features of parallelism in consistent view
 
-### Pure data parallelism
+### Data parallelism
 
-We have already known that in consistent view. The default parallelism method is data parallelism. If we choose mirrored view, we can only use data parallelism. Compare passing data in `numpy` when calling the job function (without using `flow.data.xxx_reader` in OneFlow). The difference between them is:
+We have already known that in consistent view. The default parallelism method is data parallelism. If we choose mirrored view, we can only use data parallelism. If numpy data is passed directly when the job function is called (instead of using OneFlow's `flow.data.xxx_reader` interface for data loading), the difference is that: 
 
-* In mirrored view, when we use pure data parallelism. We need to cut and reorganize data according to the number of device and use `list` to pass and receive data.
+* In mirrored view, when we use data parallelism. We need to split and reorganize data according to the number of device and use `list` to pass and receive data.
 
-* But in consistent view we have the consistency on logic. Cutting data and reorganizing data will be completed by OneFlow framework.
+* But in consistent view we have the consistency on logic. Splitting data and reorganizing data will be completed by OneFlow framework.
 
-The following figure is in consistent view, using pure data parallelism to achieve original logical network process:
+The following figure is in consistent view, using data parallelism to achieve original logical network process:
 
 ![纯数据并行](imgs/para_consistent_data.png)
 
-In pure data parallelism, we use two devices for training. Because we use **pure data parallelism**. We can see that for each original logical layer, the sample is divided in average to each device. We have complete **training model** in each device. The data after cutting processed by `op`. Finally we combine the data in each GPU and get the full complete data.
+In data parallelism, we use two devices for training. As we use **data parallelism**, we can see that for each original logical layer, the sample is divided in average to each device. We have complete **training model** in each device. The data after splitting processed by `op`. Finally we combine the data in each device and get the complete data.
```

Comment:
We have complete **training model** in each device
->
We have a complete **training model** in each device

Reply:
The data after splitting processed by `op`.
->
The data after splitting are processed by `op`.